acslib/ccure/search.py

```diff
@@ -4,29 +4,28 @@
 
 
 def left_fuzz(term):
-    return f"%{term}"
+    return f"'%{term}'"
 
 
 def right_fuzz(term):
-    return f"{term}%"
+    return f"'{term}%'"
 
 
 def full_fuzz(term):
-    return f"%{term}%"
+    return f"'%{term}%'"
 
 
 def no_fuzz(term):
-    return term
+    return f"'{term}'"
 
 
 LFUZZ = left_fuzz
 RFUZZ = right_fuzz
 FUZZ = full_fuzz
 NFUZZ = no_fuzz
 
-
 PERSONNEL_LOOKUP_FIELDS = [("FirstName", FUZZ), ("LastName", FUZZ)]
-CLEARANCE_LOOKUP_FIELDS = [("Name", FUZZ)]
+CLEARANCE_LOOKUP_FIELDS = {"Name": FUZZ, "ObjectID": NFUZZ}
```

Comment:
I definitely agree we should be consistent when this is all settled! I was thinking part of this ticket was to try out different approaches while we're still in the rough-draft phase.
Personally, the dict makes more sense to me, since each field name will only map to one function. This way also allows a (imo) simpler approach for `_compile_term`.

Reply:
I agree, this ticket and library project is indeed to try out different things, my intent with asking the question was to understand your reasoning, not to indicate  you should conform to my approach. 

I hadn't considered the automatic enforcement of a single function per entry, which I think is a really good point. And I agree I typically don't like array referencing. 

My use of a list of tuples was to "freeze" the built in function list since tuples are immutable and I think it may be important to allow implementers to extend the values they want returned.  The importance of freezing these is certainly debatable, but I think immutability has advantages when you want to rule out implementers accidentally screwing with something as they try to push your library beyond what it was designed to do.  That said, this isn't a massive concern since this isn't a stdlib sort of situation. If you're feeling the dict way, I won't complain, but we need to change the Personnel also.

Reply:
Cool, I've just changed Personnel so both searches work the same way